zkevm-circuits/src/evm_circuit/execution/sload.rs

```diff
@@ -121,16 +121,25 @@ impl<F: Field> ExecutionGadget<F> for SloadGadget<F> {
         self.callee_address
             .assign_h160(region, offset, call.address)?;
 
-        let key = block.get_rws(step, 4).stack_value();
-        let value = block.get_rws(step, 6).stack_value();
+        let mut rws = StepRws::new(block, step);
+
+        rws.offset_add(4);
+
+        let key = rws.next().stack_value();
+
+        let (_, committed_value) = rws.next().aux_pair();
+
+        let value = rws.next().stack_value();
```

Comment:
Nitpick: perhaps remove the empty lines in between `rws.next` calls (also in other places)?